_plugins/jekyll_get.rb

```diff
@@ -15,6 +15,9 @@ def generate(site)
       if !config.kind_of?(Array)
         config = [config]
       end
+      if site.config['host'] == "127.0.0.1"
```

Comment:
I typically access the site on `localhost`, not `127.0.0.1` -- or through an `ngrok` or `localtunnel` URL. Pardon my Jekyll ignorance, but how does `config['host']` get filled in?